docs/changelogs.rst

```diff
@@ -171,25 +171,32 @@ An example of how a fragment with ``release_summary`` could look like is ``chang
 Adding new Roles, Playbooks, Test and Filter Plugins
 ====================================================
 
-While ansible-core does not (yet) support playbooks, test filter plugins, and filter plugins as documentable plugins/objects, the changelog generator already has support for documenting new ones.
+Note that with ansible-core 2.11+, new roles are automatically detected if their documentation (in the argument spec) contains an appropriate ``version_added`` value. With ansible-core 2.14+ (or the current ``devel`` and ``milestone`` branches), also filter and test plugins are automatically detected if their documentation contains an appropriate ``version_added`` value. No version of ansible-core allows to document playbooks (yet), so new playbooks have to be documented as described below.
 
-This works by using special sections in changelog fragments whose names start with "`add `"::
+The following describes a way to document new playbooks. When using older versions of ansible-core during release time when the changelog is generated, this can also be used to document new roles, test plugins, and filter plugins. This works by using special sections in changelog fragments whose names start with "`add `"::
 
     ---
+    # Always needed for new playbooks:
+    add object.playbook:
+      - name: wipe_server
+        description: Wipes a server
+
+    # Only needed when ansible-core < 2.14 is used during the release process:
     add plugin.filter:
       - name: to_time_unit
         description: Converts a time expression to a given unit
       - name: to_seconds
         description: Converts a time expression to seconds
+
+    # Only needed when ansible-core < 2.14 is used during the release process:
     add plugin.test:
       - name: asn1time
         description: Check whether the given string is an ASN.1 time
+
+    # Only needed when ansible-core < 2.11 is used during the release process:
```

Comment:
Also needed if the role does not contain an argument spec with `version_added` right?

Reply:
True. Also the test/filter entries are still needed if the tests and filters have no documentation.

Reply:
41b69a80acb711e7aa14ca8cb10921e614546f0f improves this.